assignment-2-ruby/rborder/tiny_instacart.rb

```diff
@@ -0,0 +1,119 @@
+require './order_items_data'
+require 'time'
+
+# CUSTOMER APP
+
+# select a store
+def select_warehouse
+  selected_store = ""
+  stores = ['Kroger','Publix','Aldi','Sprouts']
+  stores.each do |store|
+    print "| #{store} |"
+  end
+  puts "\n"
+  while stores.include?(selected_store) == false do
+    print "\nSelect an available store: "
+    selected_store = gets.chomp
+  end
+  return selected_store
```

Comment:
fwiw ruby automatically returns the last line, so don't need to say `return selected_store`, just `selected_store`